package_manifest.yaml

```diff
@@ -1111,3 +1101,66 @@ pulpcore_packages:
     python-whitenoise: {}
     python-yarl: {}
     python-zipp: {}
+
+foreman_release_packages:
+  vars:
+    diff_package_tags:
+      - foreman-nightly-nonscl-rhel7
+      - foreman-client-nightly-el8
+      - foreman-client-nightly-rhel7
+      - foreman-client-nightly-rhel6
+      - foreman-client-nightly-rhel5
+      - foreman-client-nightly-fedora29
+    releasers:
+      - koji-foreman
+      - koji-foreman-client
+    package_base_dir: 'packages/foreman/'
+    repoclosure_config: repoclosure/yum.conf
+    repoclosure_target_repos:
+      el8:
+        - el8-foreman-client-nightly
+      rhel7:
+        - el7-foreman-nightly
+        - el7-foreman-client-nightly
+      rhel6:
+        - el6-foreman-client-nightly
+      rhel5:
+        - el5-foreman-client-nightly
+      fedora29:
+        - f29-foreman-client-nightly
+    repoclosure_lookaside_repos:
+      el8:
+        - el8-baseos
+        - el8-appstream
+        - el8-epel
+        - el8-extras
+        - el8-puppet-6
+      rhel7:
+        - el7-base
```

Comment:
Looking at the repoclosure_lookaside_repos, - el7-foreman-nightly is included.  I'm not sure if that makes a difference or not

Reply:
it probably doesnt, as output in the other job seems to be pulling in nightly

Reply:
Lets give it a shot.